presets/4.3/tune/mouseFPV_FFVCycle_Incisor_PT5_tune_filters.txt

```diff
@@ -0,0 +1,242 @@
+#$ DESCRIPTION: * RPM Filtering is recommended
+#$ DESCRIPTION: 
+#$ DESCRIPTION: ## Options:
+#$ DESCRIPTION: ### **General Options:**
+#$ DESCRIPTION: * **Dynamic Idle:** Enables Dynamic Idle for Freestyle 5"
+#$ DESCRIPTION: * **Set 93% Motor Limit:** Simulates 1800-1900kv motors down closer to 1700kv
+#$ DESCRIPTION: * **Enable Battery Sag Compensation:** Self Explanatory (see tooltip). Land at 3.5v/Cell or it's a bad time.
+#$ DESCRIPTION: * **Heavy Action Camera:** Increases the Pids/Master Slider
+#$ DESCRIPTION:
+#$ DESCRIPTION: ### **Filters:**
+#$ DESCRIPTION: * **Include RPM Filter DShot600 (F7 & Up)**: Enables RPM filtering. ESCs must support bi-directional. For F7 or better, sets Dshot600, 8k pidloop recommended (set manually).
+#$ DESCRIPTION: * **Include RPM Filter DShot300 (F4)**: Enables RPM filtering. ESCs must support bi-directional. For F4, sets Dshot300, 4k pidloop recommended (set manually). 
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Build Specs This Was Created On:
+#$ DESCRIPTION: * **Frame:** FPVCycle Incisor/PT5
+#$ DESCRIPTION: * **Motors:** FPVCycle 25mm Emerald 1880kv w/ 93 % Motor Limit 6s
+#$ DESCRIPTION: * **FC:** TMotor F7 (MPU6000)
+#$ DESCRIPTION: * **ESC:** TMotor F55a (non f3)
+#$ DESCRIPTION: * **Action Cam:** GoPro Hero 5 Session
+#$ DESCRIPTION: * **AUW:** 660-670g
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Fly Like mouseFPV | Recommendations Outside of Tune:
+#$ DESCRIPTION: * Apply mouseFPV Freestyle Rates
+#$ DESCRIPTION: * Use 250hz radio link if possible
+#$ DESCRIPTION: * **Set Jitter Reduction (feedforward_jitter_factor) to 14**
+#$ DESCRIPTION: * Set Feedforward Boost (feedforward_boost) to 10
+#$ DESCRIPTION:
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/254
+#$ WARNING: If You Choose To Include Filters Please See The Following:
+#$ INCLUDE_WARNING: misc/warnings/en/dshot.txt
+#$ FORCE_OPTIONS_REVIEW: TRUE
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# -- End Defaults --
+# -- Begin Mouse Tune --
+
+# -- PID Sliders  --
+set simplified_pids_mode = RPY
+set simplified_d_gain = 135
+set simplified_pi_gain = 100    
+set simplified_feedforward_gain = 70
+set simplified_dmax_gain = 000
+set simplified_i_gain = 090
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 115
+set simplified_master_multiplier = 110
+simplified_tuning apply
+
+# -- iTerm relax --
+set iterm_relax = RP
+set iterm_relax_type = SETPOINT
+set iterm_relax_cutoff = 10
+
+# -- TPA  --
+set tpa_rate = 75
+
+# -- Thrust linear  --
+set thrust_linear = 20
+
+
+# -- DShot Idle (default)--
+# Commonly set lower when dynamic idle is active.
+set dshot_idle_value = 400
+
+
+# -- Filters for non bi-directional setups as a base--
+
+
+# -- Gyro lowpass filters --
+# -- No Gyro Lowpass
+set gyro_lpf1_static_hz = 0
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set gyro_lpf2_static_hz = 700
+set simplified_gyro_filter = ON
```

Comment:
Idk why some people like to add the exact values and then add sliders..
But to make it easier to read the preset code, you can remove all the direct PID/Filter values affected by sliders and just use `simplified_tuning appy`

Reply:
Basically, I'm making sure gyro lpf1 is off and lpf2 is on. If I don't set the values first, the slider will leave both the dynamic LPF1 and the static lpf2 enabled and the slider will operate on both. Zeroing out LPF1 seems to flip the UI toggle for LPF1 off, and then the slider just operated the static LPF2.

I suppose the set gyro_lpf2_static_hz = 700 is redundant but the stuff on lpf1 needs to stay, unless there is a command to flip the lpf1 "toggle" in the UI off so that the slider only manipulates the lpf2? Does such a command exist?